Scripts/cmake/APP.cmake

```diff
@@ -120,9 +120,8 @@ iplug_source_tree(iPlug2_APP)
 macro(iplug_configure_app target)
   iplug_target_add(${target} PUBLIC LINK iPlug2_APP)
 
-  set(res_dir "${CMAKE_BINARY_DIR}/${PLUG_NAME}.app/Contents/Resources")
-
   if (WIN32)
+    set(res_dir "${CMAKE_BINARY_DIR}/${PLUG_NAME}.app/Contents/Resources")

```

Comment:
This looks weird for WIN32

Reply:
yep, was also a bit astonished, but this is what was set for win also before. I didn't want to change anything here, as I don't have a windows machine to test.

Reply:
i removed it for now